docs/specviz/displaying.rst

```diff
@@ -19,7 +19,31 @@ More words...
 Defining Spectral Regions
 =========================
 
-More words...
+Spectral regions can be defined by clicking on the "hammer and screwdriver" icon at the top left of an image
+viewer. Then, click the "region" icon to set the cursor dragging function in "spectral region selection" mode.
+
+.. image:: img/spectral_region_1.png
+
+Now, you can move the mouse to one of the end points (in wavelength) of the region you want to select, and drag
+it to the other end point. The selected region background will display in light gray color, and the spectral trace
+in dark red.
```

Comment:
"spectral trace in color, coded to subset number"

Reply:
Good catch!